drivers/base/null/null.c

```diff
@@ -202,7 +202,6 @@ DriverEntry(IN PDRIVER_OBJECT DriverObject,
     DriverObject->DriverUnload = NullUnload;
 
     /* Initialize the fast I/O dispatch table */
-    RtlZeroMemory(&FastIoDispatch, sizeof(FastIoDispatch));
```

Comment:
Why do you think it's redundant? Have you checked the default contents of the structure when the code is compiled with runtime checks enabled in MSVC?